CHANGELOG.md

```diff
@@ -22,9 +22,18 @@ and this project adheres to
 
 See also the [v0.107.16 GitHub milestone][ms-v0.107.15].
 
-[ms-v0.107.16]:   https://github.com/AdguardTeam/AdGuardHome/milestone/52?closed=1
--->
+[ms-v0.107.16]:   https://github.com/AdguardTeam/AdGuardHome/milestone/52?closed=
+
+### Added
+
+- The new optional `tls.override_tls_ciphers` property list, which can be set in
+  the configuration file.  It allows overriding TLS Ciphers that are used for
+  https listeners ([#4925])
 
+[#4925]: https://github.com/AdguardTeam/AdGuardHome/issues/4925
```

Comment:
New changes are added to the Unreleased section.  Please see the [KeepAChangelog recommendations](https://keepachangelog.com/en/1.0.0/).